airflow/executors/celery_executor.py

```diff
@@ -43,6 +43,10 @@ class CeleryConfig(object):
     CELERYD_CONCURRENCY = configuration.getint('celery', 'CELERYD_CONCURRENCY')
     CELERY_DEFAULT_QUEUE = DEFAULT_QUEUE
     CELERY_DEFAULT_EXCHANGE = DEFAULT_QUEUE
+    if BROKER_URL.startswith('sqla') or CELERY_RESULT_BACKEND.startswith('sqla'):
```

Comment:
Shouldn't this be `BROKER_URL` only? `CELERY_RESULT_BACKEND` is usually `db+` from what I read in the [old docs](http://docs.celeryproject.org/en/3.1/configuration.html#conf-database-result-backend)